code/modules/admin/tabs/debug_tab.dm

```diff
@@ -18,6 +18,58 @@
 	remove_verb(src, debug_verbs)
 	add_verb(src, /client/proc/enable_debug_verbs)
 
+/client/proc/enter_tree()
+	set category = "Debug.TechTree"
+	set name = "Enter Tech Tree"
+
+	if(!check_rights(R_DEBUG))
+		return
+
+	var/list/trees = list()
+
+	for(var/T in SStechtree.trees)
+		trees += list("[T]" = SStechtree.trees[T])
+
+	var/value = tgui_input_list(src, "Choose which tree to enter", "Enter Tree", trees)
```

Comment:
make it so that if there is only one tree in list `trees` then you bypass the list